libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -16,6 +16,15 @@
  *   AP_Airspeed.cpp - airspeed (pitot) driver
  */
 
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+
+#include "AP_Airspeed.h"
+
+// Dummy the AP_Airspeed class to allow building Airspeed only for all boards for plane, rover, sub, and copter & heli 2MB boards
+// This could be removed once the build system allows for vehicle_defines in header files
+// Note this Dummy construction means that a user can disable AP_Airspeed for 2MB boards but cannot enable it for 1MB boards
+#if !(APM_BUILD_COPTER_OR_HELI && BOARD_FLASH_SIZE <= 1024) && !APM_BUILD_TYPE(APM_BUILD_AntennaTracker) && !APM_BUILD_TYPE(APM_BUILD_Blimp)
```

Comment:
```suggestion
#ifndef AP_AIRSPEED_DUMMY_METHODS_ENABLED
#define AP_AIRSPEED_DUMMY_METHODS_ENABLED (!(APM_BUILD_COPTER_OR_HELI && BOARD_FLASH_SIZE <= 1024) && !APM_BUILD_TYPE(APM_BUILD_AntennaTracker) && !APM_BUILD_TYPE(APM_BUILD_Blimp))
#endif

#if AP_AIRSPEED_DUMMY_METHODS_ENABLED
```

Reply:
Done. Thank you for this! I was having the darnedest time on this one.